### DISCUSSION TITLE
TOUGH3 and PyTOUGH

### DISCUSSION THREAD

**Discussion by CUGWH** [acroucher/PyTOUGH#58] [Q&A]
Hello！
PyTOUGH documentation states that it was developed for TOUGH2 and I was wondering if it has the same applicability for TOUGH3.

HaoWei

**Comment by acroucher** [comment]
It supports the output listing files from TOUGH3, which are much the same as those from TOUGH2. On the input side, PyTOUGH doesn't yet support the extensions to the input file format introduced in TOUGH3.

**Reply by CUGWH** [reply]
Thanks for your answer.
